### PR TITLE
{data}[*/2015a] HDF5 1.8.15-patch1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/32d19abe785a477ea245 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@wpoely86: please review

File: easybuild/easyconfigs/h/HDF5/HDF5-1.8.15-patch1-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name = 'HDF5'
+version = '1.8.15-patch1'
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.15-patch1-foss-2015a.eb]: `patch1` ?

Comment (boegel) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.15-patch1-foss-2015a.eb]: it's a separate version, see https://www.hdfgroup.org/ftp/HDF5/releases/

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

File: easybuild/easyconfigs/h/HDF5/HDF5-1.8.15-patch1-foss-2015a.eb (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.15-patch1-foss-2015a.eb]: Okay, but why add both versions?

Comment (boegel) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.15-patch1-foss-2015a.eb]: I wanted to use the newer version, and just filled the gap wit 1.8.15 with foss

**Pull request merged**